layouts/_default/page-chatbot.html

```diff
@@ -0,0 +1,119 @@
+{{ define "body-class" }}
+  font-sans
+{{ end }}
+{{ define "html-class" -}}
+  scroll-smooth
+{{- end }}
+
+{{ define "main" }}
+  {{ with .Page.RenderString (.Param "takeover") }}
+    {{ . }}
+  {{ end }}
+
+
+  <section
+    class="mx-4 mb-5 flex max-w-screen-xl flex-wrap gap-6 space-y-6 px-4 pb-2 pt-12 sm:gap-x-10 md:items-start md:justify-start lg:flex-row lg:gap-x-16 xl:mx-auto"
+  >
+    <header
+      class="flex w-full flex-wrap items-baseline justify-between border-b border-black md:flex-nowrap"
+    >
+      <h2
+        class="flex-auto font-sans text-2xl font-extrabold capitalize leading-tight tracking-wide md:text-3xl"
+      >
+        {{- .Title -}}
+
+      </h2>
+      <div class="-order-1 w-full sm:order-1 sm:w-auto">
+        {{ partial "tw/breadcrumbs.html" . }}
+      </div>
+    </header>
+    {{ if or (.Content) (.Page.Params.subhed) }}
```

Comment:
This part is the same as in the regular page-landing. Maybe we should pull it out into its own partial.

Reply:
This pr is a hotfix. In the future I'd like to pull the header out into a shortcode similar to the [featured takeover ](https://www.spotlightpa.org/elections-2024/)